config/jobs/kubeflow/kubeflow-presubmits.yaml

```
@@ -83,6 +83,19 @@ presubmits:
       testgrid-dashboards: sig-big-data
       description: Presubmits for kubeflow/website.
       testgrid-num-columns-recent: '30'
+  - name: kubeflow-website-link-check
+    cluster: kubeflow
+    optional: true
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - image: docker.io/praqma/linkchecker:v9.3.1-154-g22449abb-10
```

Not an issue as this image is valid, but FYI in case you were unaware, images from docker hub do not require the `docker.io/` prefix.

 docker.io/praqma/linkchecker:v9.3.1-154-g22449abb-10
```suggestion
      - image: praqma/linkchecker:v9.3.1-154-g22449abb-10
```

---

Fixed.